### PR TITLE
Rename date methods in SearchCriteriaBuilders

### PR DESCRIPTION
**Pull request by Schwasrz94**
## Description

* rename withFromEffectiveStartDate to fromEffectiveStartDate,
  withToEffectiveStartDate to toEffectiveStartDate in
  RadiologyOrderSearchCriteria.java
* rename withFromDate to fromDate, withToDate to toDate in
  RadiologyReportSearchCriteria.java
* modify construction of RadiologyReportSearchCriteria

### CODE REVIEW

File: omod/src/main/java/org/openmrs/module/radiology/report/web/search/RadiologyReportSearchHandler.java [hunk 1]
```diff
@@ -127,21 +127,16 @@ public PageableResult search(RequestContext context) throws ResponseException {
...
-                    .includeVoided()
-                    .withStatus(status)
-                    .build();
-        } else {
-            radiologyReportSearchCriteria = new RadiologyReportSearchCriteria.Builder().withFromDate(fromDate)
-                    .withToDate(toDate)
-                    .withPrincipalResultsInterpreter(principalResultsInterpreter)
-                    .withStatus(status)
-                    .build();
+            radiologyReportSearchCriteriaBuilder = radiologyReportSearchCriteriaBuilder.includeVoided();
```
Comment (teleivo) [inline omod/src/main/java/org/openmrs/module/radiology/report/web/search/RadiologyReportSearchHandler.java]: @Schwarz94 no need to reassign is it?

Comment (teleivo) [inline omod/src/main/java/org/openmrs/module/radiology/report/web/search/RadiologyReportSearchHandler.java]: Should be possible to do

builder.includeVoided()
builder.withThisAndThat()
Criteria crit = builder.build()

**Comment by teleivo** [comment]
@Schwasrz94 please also add the class level javadoc :bowtie:

File: omod/src/main/java/org/openmrs/module/radiology/report/web/search/RadiologyReportSearchHandler.java (hunk 1 above)
Comment (Schwasrz94) [inline omod/src/main/java/org/openmrs/module/radiology/report/web/search/RadiologyReportSearchHandler.java]: True that!

**Pull request merged**